packages/common-substrate/src/project/versioned/v1_0_0/model.ts

```diff
@@ -147,6 +147,7 @@ export class ProjectManifestV1_0_0Impl
   dataSources: (SubstrateRuntimeDatasource | SubstrateCustomDatasource)[];
   @Type(() => ProjectNetworkV1_0_0)
   network: ProjectNetworkV1_0_0;
+  @IsOptional()
   @IsString()
   name: string;
```

Comment:
Should this change? Same with the other optional additions
```suggestion
  name?: string;
```